Lib/_compression.py

```diff
@@ -1,7 +1,7 @@
 """Internal classes used by the gzip, lzma and bz2 modules"""
```

Comment:
by looking the header, bz2 also seems to depending on here. could you also update it?

Reply:
We have no `bz2` rust library, unfortunately.

Reply:
Yeah, I hit the `SkipTest: No module named ' bz2'` when I copied over and ran the `test_bz2.py` as there were no changes in the `Lib/bz2.py`. Would this be connected to the [bz2.rs](https://github.com/RustPython/RustPython/blob/main/stdlib/src/bz2.rs) in `stdlib/src/`